src/vanilla/mod.rs

```diff
@@ -0,0 +1,35 @@
+use std::env;
+use std::path::PathBuf;
+
+fn var(key: &str) -> String {
+    match env::var(key) {
+        Ok(val) => val,
+        Err(err) => panic!("couldn't find env var {}; err: {:?}", key, err)
```

Comment:
When are HOME or APPDATA not set? That's weird! Also, moving the error handling to root_path means changing signature.. `io::Result` or `Option`?